borgmatic/hooks/zabbix.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import requests
+import json
+
+logger = logging.getLogger(__name__)
+
+
+def initialize_monitor(
+    ping_url, config, config_filename, monitoring_log_level, dry_run
+):  # pragma: no cover
+    '''
+    No initialization is necessary for this monitor.
+    '''
+    pass
+
+
+def ping_monitor(hook_config, config, config_filename, state, monitoring_log_level, dry_run):
+    '''
+    Ping the configured Zabbix topic. Use the given configuration filename in any log entries.
+    If this is a dry run, then don't actually ping anything.
+    '''
+
+    run_states = hook_config.get('states', ['fail'])
+
+    if state.name.lower() in run_states:
+        dry_run_label = ' (dry run; not actually pinging)' if dry_run else ''
+
+        state_config = hook_config.get(state.name.lower(),{'value': f'invalid',},)
```

Comment:
I added the spaces since I am not familiar with tox: https://github.com/borgmatic-collective/borgmatic/pull/85/commits/2877c1ad0d635324753a5ac056b9854f9dbf4d6a

Reply:
You won't need to become super familiar with it, but you'll at minimum need to be able to run tox in order to make sure that tests pass—including yours, presumably!